lib/variable/structures.cpp

```diff
@@ -25,6 +25,11 @@ make_structures<Eigen::Matrix3d, double>(const Dimensions &,
                                          const units::Unit &,
                                          element_array<double> &&);
 
+template SCIPP_VARIABLE_EXPORT Variable
+make_structures<Eigen::Affine3d, double>(const Dimensions &,
+                                         const units::Unit &,
+                                         element_array<double> &&);
```

Comment:
No definition?

Reply:
Not sure what you mean here, `make_structures` is defined in `structures.h` and this is similar to how `make_structures` is defined for martix and vector types? Have I missed some other definition that is needed?

Reply:
Sorry, I didn't see that this was just an explicit instantiation, not a declaration.